packages/tabs-overflow/src/tabs-overflow.css

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+:host {
+    inset: 0;
+    width: 100%;
+
+    --sp-tabs-overflow-next-button-right: calc(
+        -1 * var(--spectrum-component-edge-to-text-100)
+    );
+    --sp-tabs-overflow-previous-button-left: calc(
+        -1 * var(--spectrum-component-edge-to-text-100)
+    );
+    --sp-tabs-overflow-button-size: var(
+        --spectrum-global-dimension-static-size-500
+    );
+    --sp-tabs-overflow-button-height: var(--spectrum-global-dimension-size-450);
+    --sp-tabs-overflow-icon-color: var(--spectrum-gray-800);
+    --sp-tabs-overflow-shadow-color: var(--spectrum-gray-100);
+    --sp-tabs-overflow-shadow-width: 50px;
+}
+
+:host([size='s']) {
+    --sp-tabs-overflow-button-height: var(--spectrum-global-dimension-size-450);
```

Comment:
I don't remember anywhere where I found these numbers, but it looks like using `500`, `600`, etc matches the Tabs height more exactly...